src/helpers/_font-faces.scss

```diff
@@ -7,61 +7,35 @@
 
 @import "../tools/exports";
 
-/// Font Face - NTA
+/// Font Face - GOV.UK Transport
 ///
-/// Outputs the font-face declaration for NTA at the root of the CSS document
+/// Outputs the font-face declaration for GOV.UK Transport at the root of the CSS document
 /// the first time it is called.
 ///
 /// @access private
 
-@mixin _govuk-font-face-nta {
-  @include govuk-exports("govuk/helpers/font-face") {
+@mixin _govuk-font-face-govuk-transport {
+  @include govuk-exports("govuk/helpers/font-faces") {
     @at-root {
       /*! Copyright (c) 2011 by Margaret Calvert & Henrik Kubel. All rights reserved. The font has been customised for exclusive use on gov.uk. This cut is not commercially available. */
 
       @font-face {
-        font-family: "nta";
-        src: govuk-font-url("light-2c037cf7e1-v1.eot");
-        src: govuk-font-url("light-2c037cf7e1-v1.eot?#iefix") format("embedded-opentype"),
-             govuk-font-url("light-f38ad40456-v1.woff2") format("woff2"),
-             govuk-font-url("light-458f8ea81c-v1.woff") format("woff");
+        font-family: "GOVUKTransport";
+        src: govuk-font-url("light-68f0c84f0e-v2.woff2") format("woff2"),
+             govuk-font-url("light-222368e53d-v2.woff") format("woff");
         font-weight: normal;
         font-style: normal;
-        font-display: fallback;
-      }
-
-      @font-face {
-        font-family: "nta";
-        src: govuk-font-url("bold-fb2676462a-v1.eot");
-        src: govuk-font-url("bold-fb2676462a-v1.eot?#iefix") format("embedded-opentype"),
-             govuk-font-url("bold-a2452cb66f-v1.woff2") format("woff2"),
-             govuk-font-url("bold-f38c792ac2-v1.woff") format("woff");
-        font-weight: bold;
-        font-style: normal;
-        font-display: fallback;
-      }
-
-      @font-face {
-        font-family: "ntatabularnumbers";
-        src: govuk-font-url("light-tabular-498ea8ffe2-v1.eot");
-        src: govuk-font-url("light-tabular-498ea8ffe2-v1.eot?#iefix") format("embedded-opentype"),
-             govuk-font-url("light-tabular-851b10ccdd-v1.woff2") format("woff2"),
-             govuk-font-url("light-tabular-62cc6f0a28-v1.woff") format("woff");
-        font-weight: normal;
-        font-style: normal;
-        font-display: fallback;
-      }
+        font-display: swap;
+    }
 
-      @font-face {
-        font-family: "ntatabularnumbers";
-        src: govuk-font-url("bold-tabular-357fdfbcc3-v1.eot");
-        src: govuk-font-url("bold-tabular-357fdfbcc3-v1.eot?#iefix") format("embedded-opentype"),
-             govuk-font-url("bold-tabular-b89238d840-v1.woff2") format("woff2"),
-             govuk-font-url("bold-tabular-784c21afb8-v1.woff") format("woff");
+    @font-face {
+        font-family: "GOVUKTransport";
```

Comment:
`font-family: "GOV.UK Transport"`? 👼

Reply:
Not sure it's safe to have you a full stop in `font-family` 🤔 This seems to caution against things other than letters/numbers/spaces: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#Valid_family_names

Reply:
> This means that punctuation characters and digits at the start of each token must be escaped in unquoted font family names.

I think this means that punctuation is fine as long as it's with quote marks?

Reply:
Yes good spot 👍

Reply:
@aliuk2012 what do you think? We could change it based on talking to Dave later?

Reply:
@nickcolley I'd be up for changing it to 'GOVUK Transport` but lets get it through now and I'll catch up with @dashouse on Monday and see what he thinks